test/indexer/test_hca_indexer.py

```diff
@@ -1256,6 +1256,38 @@ def test_accessions_fields(self):
             self.assertEqual(['E-AAAA-00'], project['array_express_accessions'])
             self.assertEqual(['PRJNA000000'], project['insdc_study_accessions'])
 
+    def test_cell_counts(self):
+        """
+        Verify the cell counts found in project, cell_suspension, and file entities
+        """
+        # Bundles from the canned staging area, both for project 90bf705c
+        # https://github.com/HumanCellAtlas/schema-test-data/
+        bundle_fqid = self.bundle_fqid(uuid='4da04038-adab-59a9-b6c4-3a61242cc972',
+                                       version='2021-01-01T00:00:00.000000Z')
+        self._index_canned_bundle(bundle_fqid)
+        bundle_fqid = self.bundle_fqid(uuid='d7b8cbff-aee9-5a05-a4a1-d8f4e720aee7',
+                                       version='2021-01-01T00:00:00.000000Z')
+        self._index_canned_bundle(bundle_fqid)
+        hits = self._get_all_hits()
+        expected_cell_counts = {
+            'estimated_cell_count': 10000,  # 1 project
+            'total_estimated_cells': 20000 + 20000,  # 2 cell_suspensions
+            'matrix_cell_count': 2100 + 15000,  # 2 files
+        }
+        actual_cell_counts = {}
+        for hit in hits:
+            contents = hit['_source']['contents']
+            if ('projects', True) == self._parse_index_name(hit):
```

Comment:
Use tuple assignment so there is no obscure `True` literal. Also why not assert contrbutions?